app/controllers/npq_separation/admin/finance/statements/assurance_reports_controller.rb

```diff
@@ -8,7 +8,7 @@ def show
 
     respond_to do |format|
       format.csv do
-        render body: @serializer.serialize
+        send_data @serializer.serialize, filename: @serializer.filename
```

Comment:
needs a spec in `spec/requests/npq_separation/admin/finance/statements/assurance_reports_controller_spec.rb`.
should be able to check the `subject.headers["content-disposition"]`